packages/ui/v2/navigation/tabs/VerticalTabItem.tsx

```diff
@@ -71,13 +74,16 @@ const VerticalTabItem: FC<VerticalTabItemProps> = ({ name, href, tabName, info,
             <p className="text-sm font-medium leading-none">{t(name)}</p>
             {info && <p className="pt-1 text-xs font-normal">{t(info)}</p>}
           </div>
-          {isCurrent && (
+          {/* {isCurrent && (
             <div className="ml-auto self-center">
               <ChevronRight width={20} height={20} className="h-auto w-[20px] stroke-[1.5px]" />
             </div>
-          )}
+          )} */}
         </a>
       </Link>
+      {props.children?.map((child) => (
+        <VerticalTabItem key={child.name} {...child} isChild />
+      ))}
```

Comment:
Recursive nested children support

Reply:
This is super neat - Never thought about this